shared/Microsoft.Extensions.PropertyHelper.Sources/PropertyHelper.cs

```diff
@@ -503,10 +514,10 @@ private static PropertyHelper CreateInstance(PropertyInfo property)
         // Indexed properties are not useful (or valid) for grabbing properties off an object.
         private static bool IsInterestingProperty(PropertyInfo property)
         {
-            return property.GetIndexParameters().Length == 0 &&
-                property.GetMethod != null &&
+            return property.GetMethod != null &&
```

Comment:
Would extract-local on `property.GetMethod` help here? It's accessed 4 times in a row.